ucp/_libs/ucx_api.pyx

```diff
@@ -386,6 +388,84 @@ cdef class UCXWorker(UCXObject):
         assert_ucs_status(status)
         return UCXEndpoint(self, <uintptr_t>ucp_ep)
 
+    cpdef ucs_status_t fence(self) except *:
+        cdef ucs_status_t status = ucp_worker_fence(self._handle)
+        assert_ucs_status(status)
+        return status
+
+    def flush(self, cb_func, cb_args, cb_kwargs=dict()):
+        cdef ucs_status_ptr_t req
+        cdef ucp_send_callback_t _send_cb = <ucp_send_callback_t>_send_callback
+
+        cdef ucs_status_ptr_t status = ucp_worker_flush_nb(self._handle, 0, _send_cb)
+        return _handle_status(
+            status, 0, cb_func, cb_args, cb_kwargs, 'flush', self._inflight_msgs
+        )
+
+    def get_address(self):
+        return UCXAddress(self)
+
+
+def _ucx_address_finalizer(uintptr_t handle_as_int, UCXWorker worker):
+    cdef ucp_address_t *address = <ucp_address_t *>handle_as_int
+    cdef ucp_worker_h ucp_worker = <ucp_worker_h><uintptr_t>worker.handle
```

Comment:
Wouldn't it make sense to just pass `worker_handle_as_int` here to make both arguments type-uniform with `uintptr_t` and remove the need for a double cast here?

Reply:
Good point. Yeah there is a faster way to access this.

```suggestion
    cdef ucp_worker_h ucp_worker = worker._handle
```

Reply:
Got ya, will add test described above and change up how the handle is handled.

Reply:
Thought this was resolved, but it appears the diff was just outdated. Mentioned in a new review comment below ( https://github.com/rapidsai/ucx-py/pull/594#pullrequestreview-502316610 ).